.env

```diff
@@ -15,7 +15,7 @@ _APP_OPTIONS_ROUTER_PROTECTION=disabled
 _APP_OPTIONS_FORCE_HTTPS=disabled
 _APP_OPTIONS_FUNCTIONS_FORCE_HTTPS=disabled
 _APP_OPENSSL_KEY_V1=your-secret-key
-_APP_DOMAIN=localhost
+_APP_DOMAIN=traefik
```

Comment:
Yes, domain goes to function as env variable. In case of Cloud that would be `cloud.appwrite.io`. We need this here so test pass - otherwise test using ENV variable for endpoint API will not pass